docs/readthedocs/models/input/participant/bm.md

```diff
@@ -0,0 +1,40 @@
+(bm-model)=
+
+# Biomass plant
+
+Model of a biomass power plant.
+
+## Attributes, Units and Remarks
+
+### Type Model
+
+| Attribute           | Unit    | Remarks                                                   |
+| ------------------- | ------- | --------------------------------------------------------- |
+| uuid                | --      |                                                           |
+| id                  | --      | Human readable identifier                                 |
+| capex               | €       | Capital expenditure to purchase one entity of this type   |
+| opex                | € / MWh | Operational expenditure to operate one entity ofthis type |
+| activePowerGradient | % / h   | Maximum permissible rate of change of power               |
+| sRated              | kVA     | Rated apparent power                                      |
+| cosphiRated         | --      | Rated power factor                                        |
+| etaConv             | %       | Efficiency of the assets inverter                         |

```

Comment:
There's some grid tables left, like this one.  Is it possible to convert these as well?

Reply:
Everything should be corrected now

Reply:
But not. The Links in the new listtables don't work.